README.md

```diff
@@ -59,7 +59,8 @@ To build the project, see the [Contributing guide](CONTRIBUTING.md#Build) :sligh
 
 # License
 
-`bpmn-visualization` is released under the `Apache 2.0` license.
+`bpmn-visualization` is released under the [Apache 2.0](LICENSE) license. \
```

Comment:
:question: could we put this on the same line?

Reply:
It's possible. But I find it less readable.

Reply:
ok, let's keep your initial proposal then :grinning: